ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -346,6 +348,7 @@ objects:
             region: ${AWS_REGION}
             userTags:
               expirationDate: ${EXPIRATION_DATE}
+              jobSpec: ${JOB_SPEC}
```

Comment:
Because many resource are created via CloudFormation for this UPI flow (instead of via the installer and Terraform), we'll also want to include the job spec [here][1].

[1]: https://github.com/openshift/release/blob/3bbc5c72bd9d2b3c28945c1f654d272ea10cb2f1/ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml#L419